recipes/zeus_expected/all/conandata.yml

```
@@ -0,0 +1,10 @@
+sources:
+  "1.1.0":
+    url: "https://github.com/zeus-cpp/expected/archive/refs/tags/v1.1.0.tar.gz"
+    sha256: "a963eba43f227498da2cbb924265344209696320c75ee63a92073936bb49f7e5"
+
+patches:
+  "1.1.0":
+    - patch_file: "patches/0001-fix-compiler-errors-with-msvc-v142.patch"
```

```suggestion
    - patch_file: "patches/0001-fix-compiler-errors-with-msvc-v142.patch"
      patch_source: "https://github.com/zeus-cpp/expected/pull/7/commits/82ed513aa65a3c2c7b8f8af7c63e0b8b57637dfb"
```

Try to keep source of the change tracked